frontend/templates/troubleshooting/index.tsx

```diff
@@ -905,7 +907,18 @@ function AnswersCTA({ answersUrl }: { answersUrl: string }) {
    );
 }
 
-function RelatedProblems({ problems }: { problems: Problem[] }) {
+function RelatedProblems({
+   wikiData,
+   hasRelatedPages,
+}: {
+   wikiData: TroubleshootingData;
+   hasRelatedPages?: boolean;
+}) {
+   const { linkedProblems } = wikiData;
+   const { displayTitle, imageUrl, viewUrl, description } = wikiData.category;
+
+   const problemCount = linkedProblems.length;
+   const isMobile = useBreakpointValue({ base: true, sm: false });
    const bufferPx = useBreakpointValue({ base: -40, lg: 0 }, { ssr: false });
```

Comment:
As far as I'm aware, using breakpoints like this effectively means there is a FUC because the server-side doesn't know the client's width and thus always returns the 'base' value. After client side hydration the JS is run and the real value is returned (effectively, on the second render).

It's better to use https://chakra-ui.com/docs/styled-system/responsive-styles which results in rendering css using media queries instead.

Reply:
Looks like this might be the one to use: https://chakra-ui.com/docs/hooks/use-media-query

Reply:
[`47e4f37`](https://github.com/iFixit/react-commerce/pull/2009/commits/47e4f37c110841ae0b0723ab210b2479d88131c9) I found https://chakra-ui.com/docs/hooks/use-media-query